app/controllers/mgames_controller.rb

```diff
@@ -15,13 +16,23 @@ def create
         end
       
         def update
+          mgame = Mgame.find(params[:id])
+          if mgame.update(mgame_params)
+            render json: mgame
+          else
+            render json: mgame.errors, status: :unprocessable_entity
+          end
         end
       
         def destroy
+          mgame = Mgame.find(params[:id])
+          mgame.destroy
+          render json: mgame
```

Comment:
Very valid approach. I would recommend you consider sending something a bit more meaningful. You can send  all the cats back to show that the entry was deleted.
here is how that would look.
```js
  def destroy
    cats = Cat.all
    cat = Cat.find(params[:id])
    if cat.destroy
      render json: cats
    else  
      render json: cat.errors, status: 422
    end
  end
```